pallets/phala/src/registry.rs

```diff
@@ -74,6 +79,13 @@ pub mod pallet {
 	#[pallet::storage]
 	pub type BenchmarkDuration<T: Config> = StorageValue<_, u32>;
 
+	/// Allow list of PRuntime
+	///
+	/// Only pRuntime within the list can do register.
+	#[pallet::storage]
+	#[pallet::getter(fn pruntime_allowlist)]
+	pub type PRuntimeAllowList<T> = StorageValue<_, Vec<PRuntimeHash>>;
```

Comment:
Don't introduce another type alias `PRuntimeHash`. It's ok to just use `Vec<Vec<u8>>`